### ISSUE TITLE
hs.screen:setMode() new params not documented

### ISSUE THREAD

**Issue by rsefer** [Hammerspoon/hammerspoon#2756]
https://github.com/Hammerspoon/hammerspoon/commit/bcda1cf166cbcda93031262ddcae7d9ac500f75c added 2 params to `hs.screen:setMode()` that are not documented. When trying to call `setMode` with only 3 params (width, height, scale) I frequently get the following error:

```
ERROR: incorrect type 'no value' for argument 5 (expected number)
````

In looking at `extensions/screen/internal.m` it seems the 2 additional params are frequency [4] and depth [5]. When I set those explicitly, it seems to work.

TL;DR:
`hs.screen:setMode(1792, 1120, 2)` no longer works
`hs.screen:setMode(1792, 1120, 2, 60, 8)` seems to work, but is undocumented

**Comment by rsefer** [comment]
Appears to be fixed.

**Comment by nielsbom** [comment]
I just had this problem too after updating MacOS to 11.4. Still needed to fix it by passing all required args.
